doc/plugin_tutorial/tuto0/src/g_tuto0.mlg

```diff
@@ -7,12 +7,75 @@ open Ltac_plugin
 
 }
 
+(*** Printing messages ***)
+
+(*
+ * This defines a command that prints HelloWorld.
+ * Note that Feedback.msg_notice can be used to print messages.
+ *)
 VERNAC COMMAND EXTEND HelloWorld CLASSIFIED AS QUERY
 | [ "HelloWorld" ] -> { Feedback.msg_notice (strbrk Tuto0_main.message) }
 END
 
+(*
+ * This is a tactic version of the same thing.
+ *)
 TACTIC EXTEND hello_world_tactic
 | [ "hello_world" ] ->
   { let _ = Feedback.msg_notice (str Tuto0_main.message) in
     Tacticals.New.tclIDTAC }
 END
+
+(*** Printing warnings ***)
+
+(*
+ * This defines a command that prints HelloWorld as a warning.
+ * Note that Feedback.msg_warning can be used to print warnings.
+ *)
+VERNAC COMMAND EXTEND HelloWarning CLASSIFIED AS QUERY
+| [ "HelloWarning" ] ->
+   {
+     let warn =
+       CWarnings.create
```

Comment:
This is wrong, cwarnings.create should be called at the toplevel, not by the command.
ie
~~~ocaml
let tuto_warn = CWarnings.create ~name:"bla" ~category:"bla" Pp.(fun () -> str ""say something")

VERNAC COMMAND EXTEND HelloWarning CLASSIFIED AS QUERY
| [ "HelloWarning" ] -> { tuto_warn () }
END
~~~

Reply:
and call the same `tuto_warn` in the tactic.